test/Recipe-test.js

```diff
@@ -3,10 +3,10 @@ import Ingredient from '../src/classes/Ingredient';
 import Recipe from '../src/classes/Ingredient';
 
 describe('Recipe Class', function() {
-  let recipeData, recipe;
+  let recipeDetails, recipe;
 
   beforeEach(function() {
-    recipeDatum = {
+    recipeDetails = {
```

Comment:
it may make the file shorter to do this (instead of putting the actual object literals in):

```
import recipes from '../src/data/recipes.js';

recipe = new Recipe(recipes[0])
```